Kernel/Process.cpp

```diff
@@ -4011,14 +4011,18 @@ int Process::sys$mount(const Syscall::SC_mount_params* user_params)
     if (!validate_read_and_copy_typed(&params, user_params))
         return -EFAULT;
 
-    auto source = validate_and_copy_string_from_user(params.source);
+    auto source_fd = params.source_fd;
     auto target = validate_and_copy_string_from_user(params.target);
     auto fs_type = validate_and_copy_string_from_user(params.fs_type);
 
-    if (source.is_null() || target.is_null() || fs_type.is_null())
+    if (target.is_null() || fs_type.is_null())
         return -EFAULT;
 
-    dbg() << "mount " << fs_type << ": source " << source << " @ " << target;
+    auto fd = file_description(source_fd);
```

Comment:
These variables are conventionally called `description` whereas `fd` is used for file *descriptors* (`int`)